src/main/java/com/blackducksoftware/integration/jira/web/controller/AccessConfigController.java

```diff
@@ -49,7 +49,7 @@
 @Path("/config/access")
 public class AccessConfigController extends ConfigController {
     // This must be "package protected" to avoid synthetic access
-    final BlackDuckJiraLogger logger = new BlackDuckJiraLogger(Logger.getLogger(this.getClass().getName()));
+    private final Logger logger = LoggerFactory.getLogger(getClass());
```

Comment:
If we are going with the synthetic access approach, then we should remove the comment.